v3/src/components/axis/helper-models/axis-helper.ts

```diff
@@ -0,0 +1,92 @@
+import { select } from "d3"
+import { isVertical } from "../../axis-graph-shared"
+import { AxisBounds, AxisPlace, axisPlaceToAxisFn } from "../axis-types"
+import { IAxisModel } from "../models/axis-model"
+import { IDataDisplayContentModel } from "../../data-display/models/data-display-content-model"
+import { IAxisLayout } from "../models/axis-layout-context"
+import { MultiScale } from "../models/multi-scale"
+
+export interface IAxisHelperArgs {
+  displayModel: IDataDisplayContentModel
+  subAxisIndex: number
+  subAxisElt: SVGGElement | null
+  axisModel: IAxisModel
+  layout: IAxisLayout
+  isAnimating: () => boolean
+}
+
+export class AxisHelper {
+  displayModel: IDataDisplayContentModel
+  subAxisIndex: number
+  subAxisElt: SVGGElement | null
+  axisModel: IAxisModel
+  layout: IAxisLayout
+  isAnimating: () => boolean
+  multiScale: MultiScale | undefined
+  axisPlace: AxisPlace
```

Comment:
Instead of a property, you could also just make this an accessor function which forwards to the model.

Reply:
Done